content/en/hosting/4.x/app-developer.md

```diff
@@ -242,17 +249,75 @@ COUCHDB_DATA=${HOME}/cht_4_app_developer-dir/couch
 CHT_COMPOSE_PATH=${HOME}/cht_4_app_developer-dir/compose
 CHT_NETWORK=cht_4_app_developer
 EOF
-```
+{{< /tab >}}
+{{< tab header="Multi-Node CouchDB" >}}
+cat > ~/cht_4_app_developer-dir/.env << EOF
+NGINX_HTTP_PORT=8080
+NGINX_HTTPS_PORT=8443
+COUCHDB_USER=medic
+COUCHDB_PASSWORD=password
+CHT_COMPOSE_PROJECT_NAME=cht_4_app_developer
+DOCKER_CONFIG_PATH=${HOME}/cht_4_app_developer-dir
+COUCHDB_SECRET=19f3b9fb1d7aba1ef4d1c5ed709512ee
+COUCHDB_UUID=e7122b1e463de4449fb05b0c494b0224
+COUCHDB_DATA=${HOME}/cht_4_app_developer-dir/couch
```

Comment:
This is not used in multi-node, data is saved in `DBX_DATA`

Reply:
ah - thank you!

when testing this still works, I found some hard coded `mrjones` paths in there which I tidied up as well just now